### PR TITLE
Added GDS/Gov banner and alpha phase

### PR DESCRIPTION
**Pull request by danielchildsbjss**
Also removed sidemenu for now (not part of core journey).

image

### CODE REVIEW

File: src/theme/sass-partials/_sidebar.scss [hunk 1]
```diff
@@ -49,36 +43,6 @@
...
-    background: map-get($colors, lighterGrey);
-    height: 57px;
-    color: map-get($colors, textGrey);
-    line-height: 55px;
-    span {
-      margin-left: 20px;
-    }
-  }
-}
-
```
Comment (ataylorBJSS) [inline src/theme/sass-partials/_sidebar.scss]: Can we remove this file completely, now we no longer have the sidebar?  I think the footer styles could definitely go.

File: src/components/components.module.ts [hunk 2]
```diff
@@ -4,19 +4,25 @@ import { IonicModule } from 'ionic-angular';
...
     JournalHeaderComponent,
+    HeaderComponent,
     TestSummaryComponent,
     ButtonCardComponent,
     TestSummarySectionComponent
   ],
-  imports: [IonicModule.forRoot(PageHeaderComponent)],
+  imports: [
+    IonicModule.forRoot(PageHeaderComponent)
+  ],
```
Comment (ataylorBJSS) [inline src/components/components.module.ts]: Did the prettier plugin change this? Using the current config, it should be all on one line.

Comment (danielchildsbjss) [reply src/components/components.module.ts]: No - I did, making the arrays consistent...I can change it back!

Comment (ataylorBJSS) [reply src/components/components.module.ts]: no worries, I think it's fine. I was just wondering if prettier is actually doing what it should be for everyone. We can change the config if this is the format we want.

**Pull request merged**